config/cinder/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ ceph_cinder_backup_cache_mode: "{{ ceph_cache_mode }}"
 
 # Due to Ansible issues on include, you cannot override these variables. Please
 # override the variables they refernce instead.
-cinder_pool_name: "{{ ceph_cinder_pool_name }}"
+cinder_pool_name: "rbd"
```

Comment:
this should be renamed to ceph_cinder_pool_name as well as other similar values.